### DISCUSSION TITLE
[BUG]Remove-PnPFile -ServerRelativeUrl is not delete the file on online sharepoint. it returns no error.

### DISCUSSION THREAD

**Discussion by ziadfeghali** [pnp/powershell#3731] [General]
Remove-PnPFile -ServerRelativeUrl is not deleting the file on online sharepoint. it returns no error.
i have the vesion 1.10 that was working for so many years and lately it stop deleting. i tried to use many version of it, latest, latest -1, 1.12 and all return with the same symptom. no error, the file is not deleted on online sharepoint.
Please advise if you nee more info.

**Comment by rasmussjostrom** [comment]
I have the same issue, did you find a resolution for this?

**Comment by M365CraftsMan** [comment]
I have the exact same issue
Has someone been able to look this up?

**Comment by Macro21** [comment]
Be carefull with "Server" and "Site" relative url:

Remove-PnPFile -ServerRelativeUrl "/sites/miSitio/Documentos/InformeAnual.xlsx" -Recycle
Remove-PnPFile -SiteRelativeUrl "/Documentos/InformeAnual.xlsx" -Recycle
